tools/rsdl/syntax-highlighting/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,29 @@
+# Welcome to your VS Code Extension
```

Comment:
Actually need to test that as well. at some point extension can be added to marketplace and documented on website.